packages/web/src/store/application/ui/withdraw-usdc/sagas.ts

```diff
@@ -203,8 +210,52 @@ function* doWithdrawUSDC({ payload }: ReturnType<typeof beginWithdrawUSDC>) {
         )
       }
     }
-    // TODO: transfer USDC to destination associated token account
+    // TODO: math.min(amount, balance)
```

Comment:
do we need to have this check here? ideally UI will handle this

Reply:
I think we do bc there might be slightly less USDC after swapping for fees. will open ticket tho.